components/ingest-service/pipeline/processor/corrections_run.go

```diff
@@ -0,0 +1,37 @@
+package processor
```

Comment:
There is no code change here. This code was moved into its own file from the transmogrify_run.go file.